docs/cluster.md

```diff
@@ -15,20 +15,25 @@ Currently, Jellyfish doesn't offer an option to split a room across multiple mac
 
 ## Configuring a Cluster
 
-Jellyfish cluster can be created in 4 simple steps:
-
+Currently Jellyfish supports two clustering strategy `DNS` and `NODES_LIST`.
+Configuration for each strategy is described below.
+But before that you need to set two flags:
 1. Enable distribution mode with `JF_DIST_ENABLED=true`
-1. Give your node a name with `JF_DIST_NODE_NAME`
-1. Specify a list of nodes to connect to with `JF_DIST_NODES`
-1. Configure HTTP and metrics ports so they don't overlap with other nodes.
-You can do this with `JF_PORT` and `JF_METRICS_PORT` environment variables.
+2. (Optionally) Define which distribution strategy will be used with `JF_DIST_STRATEGY_NAME`. <br/>
+By default it is setup to `NODES_LIST`.
 
 :::tip Distribution Environment Variables
 
 List of all cluster-related environment variables is available [here](./getting_started/installation#distribution).
 
 :::
 
+:::tip Configure HTTP and metrics ports so they don't overlap with other nodes.
+
+You can do this with `JF_PORT` and `JF_METRICS_PORT` environment variables.
+
+:::
```

Comment:
This is more related to specific examples and we already mention this in `Running from source` for NODES_LIST strategy and have correct docker-compose examples so I would remove this?